sources/api/apiclient/src/apply.rs

```diff
@@ -2,15 +2,19 @@
 //! TOML settings files, in the same format as user data, or the JSON equivalent.  The inputs are
 //! pulled and applied to the API server in a single transaction.
 
+use aws_sdk_s3::Client;
+use crate::uri_resolver::{StdinUri, FileUri, HttpUri, S3Uri, UriResolver};
+use crate::apply::error::{UriSnafu, NoResolverSnafu};
 use crate::rando;
 use futures::future::{join, ready, TryFutureExt};
 use futures::stream::{self, StreamExt};
 use reqwest::Url;
 use serde::de::{Deserialize, IntoDeserializer};
 use snafu::{futures::try_future::TryFutureExt as SnafuTryFutureExt, OptionExt, ResultExt};
-use std::path::Path;
+use std::{convert::TryFrom, path::Path};
```

Comment:
nit: I prefer this format for std lib imports:

```suggestion
use std::convert::TryFrom;
use std::path::Path;
```

Reply:
fixed